src/roles/bodyguard.py

```diff
@@ -1,22 +1,26 @@
+from __future__ import annotations
+
 import re
 import random
 import itertools
 import math
 from collections import defaultdict
+from typing import Set
 
 import src.settings as var
 from src.utilities import *
-from src import users, channels, debuglog, errlog, plog
+from src import channels, debuglog, errlog, plog
 from src.functions import get_players, get_all_players, get_target, get_main_role
 from src.decorators import command, event_listener
 from src.containers import UserList, UserSet, UserDict, DefaultUserDict
 from src.messages import messages
 from src.status import try_misdirection, try_exchange, add_protection, add_dying
 from src.cats import Wolf
+from src.users import User
 
-GUARDED = UserDict() # type: UserDict[users.User, users.User]
+GUARDED: UserDict[User, User] = UserDict()
 PASSED = UserSet()
-DYING = set()
+DYING: Set[User] = set()
```

Comment:
Should be a UserSet() and additionally cleared in the reset event. Not sure why this wasn't the case already